lib/numbers.md

```diff
@@ -5,20 +5,44 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Float (decimal numbers) and Fixnum (integers)
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Operations: addition (+), subtraction (-), multiplication (*), to the power (**), division (/)
+            modulo (%)
+
+Comparisons : greater than (>), greater than or equal to (>=), less than (<), 
+              less than or equal to (<=)
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+the '+' operation on strings will "add" the strings together by combining them, the output is still a string
+     example: "23" + "dog"   #=>   "23dog"
+the '+' operation on numbers will add the numbers together and output the sum
+    example: 21 + 15    #=> 36
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+Since "20" is a string (-) and (/) operations will not work on it at all, if you try either of these
+it will produce an error. (+) will work if another string is being "added" to it, otherwise
+it causes an error because you cannot add a string and a number together. However, (*) will work, 
+but the output will still be a string, not a number.
+    Examples:  "20" * 3     #=> "202020"
+                "20" + 2    #=> error
+                "20" + "2"  #=> "202"
+ 
+To be able to perform a mathematical operation you would have to first convert the 
+string to a number: "20".to_i   (converts 20 to an integer)
```

Comment:
Your answer would not be very precise because when dividing two integers the return will be another integer, it does not account for any remainders. 20/3 = 6 in ruby. To get a more precise answer you would need to change at least one of the integers to a float. So if you had num = "20".to_f and divided num by 3, you would get 6.6666667